lib/handlers/accessRequest/accessRequestV1.js

```diff
@@ -8,6 +8,7 @@ const process = async (config, logger, data) => {
     approved: data.approved,
     reason: data.reason,
     helpUrl: `${config.notifications.helpUrl}/contact`,
+    interactionsUrl:`${config.notifications.interactionsUrl}`,
```

Comment:
I think this url should be services instead. So config.notifications. servicesUrl